examples/lib/count-star-height.js

```diff
@@ -0,0 +1,83 @@
+/* Author: Jamie Davis <davisjam@vt.edu>
```

Comment:
ditto, please use standard doc header with license, and copyright (and you can keep your name if you'd like)